### ISSUE TITLE
Status of the connection to DS2321 and temperature question

### ISSUE THREAD

**Issue by Saur0o0n** [hideakitai/DS323x#2]
Hi,

Is there any (good) way to check if the DS3231 is actually connected? What units is used for temperature - I got results around 1500-1600 in room temperature (22C).

**Comment by hideakitai** [comment]
Hi, 

> Is there any (good) way to check if the DS3231 is actually connected? 

Currently there is no function to check it, but you can confirm it by using function like this:

``` C++
bool isI2CDeviceConnected(const uint8_t addr) {
    Wire.beginTransmission(addr);
    return Wire.endTransmission() == 0;
}
```

It may be good to return `bool` from `attach()` to check if the connection to DS3231 succeed.



>  What units is used for temperature - I got results around 1500-1600 in room temperature (22C).

Sorry maybe this is fixed in v0.1.2 (I've pushed to master but forgot to release...).
Please try it again in latest v0.1.2.

**Comment by Saur0o0n** [comment]
Hi,
Both check function and temperature are working fine now - thanks!
